### ISSUE TITLE
WP_CLI - Question

### ISSUE THREAD

**Issue by benlumia007** [WordPress/wordpress-importer#141]
Is it possible to wrap the following echo when the import is completed
from
```
echo '<p>' . __( 'All done.', 'wordpress-importer' ) . ' <a href="' . admin_url() . '">' . __( 'Have fun!', 'wordpress-importer' ) . '</a>' . '</p>';
echo '<p>' . __( 'Remember to update the passwords and roles of imported users.', 'wordpress-importer' ) . '</p>';
```
to
```
if ( ! defined( 'WP_CLI' ) ) {
    echo '<p>' . __( 'All done.', 'wordpress-importer' ) . ' <a href="' . admin_url() . '">' . __( 'Have fun!', 'wordpress-importer' ) . '</a>' . '</p>';
    echo '<p>' . __( 'Remember to update the passwords and roles of imported users.', 'wordpress-importer' ) . '</p>';
}
```
I found that having to use WP CLI to import data and each time when it completes it displays that even if you set import process with `--quiet`

the echo should still remain on output if you are importing from the dashboard. 

What you guys think!

**Comment by jrfnl** [comment]
Not a guy here, but here's my opinion:

1. I think the "Remember to update the passwords..." text is still pretty valuable when importing via WP-CLI, but it shouldn't display as HTML in that case, so a toggle to display (some of) the text depending on the context in which the comamnd was run may be appropriate.
2.  I imagine it should be doable to figure out if WP-CLI is run in `quiet` mode ? And possibly not display the message in that case ?

**Comment by benlumia007** [comment]
Right, I agree with what you said with number 1, it shouldn't display as HTML  and for Number 2, having to use `--quiet` should also prevent from displaying. So this is what it looks like when using WP CLI to import data
```
benlumia007@arcturus:~/.dev$ sturdydocker up
benlumia007@arcturus:~/.dev$ sturdydocker provision
<p>All done. <a href="https://dev.wordpress.test/wp-admin/">Have fun!</a></p><p>Remember to update the passwords and roles of imported users.</p>
benlumia007@arcturus:~/.dev$ 
```
this is with using `wp import something.xml --authors=create --quiet` so we should see if we can find way to not display the html tag when using the `--quiet` tag.

other than that without the --quiet tag, then it should display the messages when import is completed.

**Comment by pattonwebz** [comment]
What about a `WP_CLI::log()` message to inform about the password changes and any other relevant info? So it's not piping echo commands out but is still informing the useful information in a way expected by WP_CLI?
